test/code/plugins/oms_common_test.rb

```diff
@@ -345,5 +345,16 @@ def test_get_current_timezone_symlink
       # sanity check
       assert_equal('Pacific Standard Time', timezone, "timezone is not expected")
     end
+    
+    def test_get_current_timezone_relative_symlink
+      $log = MockLog.new
+      File.delete(@tmp_localtime_file.path)
+      File.symlink('/usr/../usr/share/zoneinfo/right/America/Los_Angeles', @tmp_localtime_file.path)
```

Comment:
You don't need /usr/../usr/share/zoneinfo/right/America/Los_Angelese

Take off the leading `/usr/..`, as it serves no (obvious) purpose